### PR TITLE
Distribute debug build on firebase

### PR DESCRIPTION
**Pull request by lutangar**
> This only run when master is updated 
1. Bundle the app
2. Upload it to Firebase

I suppose I'll be able to download it back to upload it on Github for example...
Have a look at main.yml and tell me what's missin'

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+name: Disribute debug build
```
Comment (felix-lambert) [inline .github/workflows/main.yml:1]: "Distribute ..."

**Review by felix-lambert — APPROVED** [review]

**Comment by lutangar** [comment]
@JalilArfaoui I added you as a reviewer, I'd like to hear your opinion on this.

From Firebase doc, it's possible to fill release notes :
https://firebase.google.com/docs/app-distribution/android/distribute-cli

But at the moment I suppose the version number is incremented manually and something as semantic-release would be helpful to automate this.

**Comment by JalilArfaoui** [comment]
@lutangar yep, this sounds like a good start:

And I confirm that version number is incremented manually for the time being …

Can you pass this «Ready for review» ?

**Comment by lutangar** [comment]
Done

**Comment by lutangar** [comment]
> As discussed with @JalilArfaoui still left to change the trigger event to a "tag" event

**Review by JalilArfaoui — APPROVED** [review]

**Pull request merged**